cartography/intel/github/util.py

```diff
@@ -81,12 +80,12 @@ def call_github_api(query: str, variables: str, token: str, api_url: str) -> Dic
 
 
 def fetch_page(
-        token: str,
-        api_url: str,
-        organization: str,
-        query: str,
-        cursor: Optional[str] = None,
-        **kwargs: Any,
+    token: str,
+    api_url: str,
+    organization: str,
+    query: str,
+    cursor: Optional[str] = None,
+    **kwargs: Any,
```

Comment:
What's the current convention in this repo? 

Maybe we should set up `black` on this repo. 
Changes like this can confuse the git history.